test/khash_test.erl

```diff
@@ -0,0 +1,443 @@
+-module(khash_test).
+
+-compile([export_all]).
+
+-define(NODEBUG, true).
```

Comment:
But only when the tests ran after midnight... Just kidding.

NODEBUG implies NOASSERT only if testing is not enabled and rebar always compiles the tests with `-DTEST`. So NODEBUG here acts as a turn off for debugging macros only.